### ISSUE TITLE
Add Ginga-HTML5 support for the document loader

### ISSUE THREAD

**Issue by rafael2k** [TeleMidia/ginga#152]
As per ABNT NBR 15606-10, Ginga-HTML5 is a new Ginga subsystem, and it should be support by Ginga profile D receivers (together with Ginga-NCL subsystem).

**Comment by Ronkiro** [comment]
bump, i think we'll need this ready for TV 3.0

**Comment by alanlivio** [comment]
I totally agree that should be supported. Just to remember what has been done in this:

- first version when having Directfb backend: use ncurses
- second version when having SDL backend: use Berkelium (chrome-based)
- last version uses Cairo-gst backend: uses Chromium embedded by [PlayerHTML.cpp](https://github.com/TeleMidia/ginga/blob/master/lib/PlayerHTML.cpp) but is currently, not stable.

Other possible approaches are:
- use Webkit inside Cairo context
- use Webkit inside GST (nclua standalone proposed by gflima)
